PetGame/Controllers/UserController.cs

```diff
@@ -76,9 +69,9 @@ public IActionResult PostPet([FromForm, FromBody] CreatePetModel model)
                 // HACK: Throw the new pet id into the query string just so we know that an id was added
                 return Redirect($"/?PetId={pet.PetId}");
```

Comment:
Either this diff is really broken or you are doing something seriously wrong here.